src/services/playlistVideos_services.js

```diff
@@ -0,0 +1,102 @@
+import { fetchApi } from "../shared/utils/fetchApi";
+
+export const watchLaterFetchApi = async () => {
+  return await fetchApi({
+    url: "/api/user/watchlater",
+    method: "get",
+  });
+};
+
+export const addToWatchLaterApi = async (video) => {
+  return await fetchApi({
+    url: "/api/user/watchlater",
+    method: "post",
+    body: { video },
+  });
+};
+
+export const deleteFromWatchLaterApi = async (videoId) => {
+  return await fetchApi({
+    url: `/api/user/watchlater/${videoId}`,
+    method: "delete",
+  });
+};
+
+export const playlistVideosFetchApi = async () => {
+  try {
+    const promise1 = await fetchApi({
+      url: "/api/user/watchlater",
+      method: "get",
+    });
+    const promise2 = await fetchApi({
+      url: "/api/user/likes",
+      method: "get",
+    });
+    const promise3 = await fetchApi({
+      url: "/api/user/history",
+      method: "get",
+    });
+    const promise4 = await fetchApi({
+      url: "/api/user/playlists",
+      method: "get",
+    });
+
+    const response = await Promise.all([promise1, promise2, promise3,promise4]);
+    console.log(response)
+    return {
+      data: {
+        watchLater: response[0].data.watchlater,
+        likes: response[1].data.likes,
+        history: response[2].data.history,
+        playlists : response[3].playlists
```

Comment:
You can destructure the response in line `44` and use it directly in data rather than saying `response[0]` etc..